test/test-utils/helpers/moveHelper.ts

```diff
@@ -197,4 +200,36 @@ export class MoveHelper extends GameManagerHelper {
      */
     await this.game.phaseInterceptor.to("EnemyCommandPhase");
   }
+
+  /**
+   * Emulate a player's target selection after a move is chosen, called automatically by {@linkcode MoveHelper.select}.
+   * Will trigger during the next {@linkcode SelectTargetPhase}
+   * @param movePosition The index of the move in the Pokemon's {@linkcode Pokemon.moveset | moveset}
+   * @param targetIndex The index of the attack target, or `undefined` for multi-target attacks
```

Comment:
```suggestion
   * @param movePosition - The index of the move in the Pokemon's {@linkcode Pokemon.moveset | moveset}
   * @param targetIndex - The index of the attack target, or `undefined` for multi-target attacks
```
Might as well update this at the same time.